### ISSUE TITLE
source version and displayed version should match

### ISSUE THREAD

**Issue by agaida** [andrew-bibb/cmst#112]
not a big problem - but help/info say, cmst is 15.$foo.$bar, the tar balls are 2015.$foo.$bar - so i suggest to change this in apps/ressources.h to 2015.

What do you think?

**Comment by andrew-bibb** [comment]
I agree.  Not sure what I used in the change logs but I'll make sure those match as well.

There was a previous issue/suggestion to convert to semantic versioning, but to me dates seem to make so much sense.  Used to make more sense when I could remember about what part of the year it was I was working on a specific thing, but this project has gotten legs beyond what I expected and I've kind of lost track of what I worked on when.  If you think semantic would work better (and it actually might) we could make the clean version numbering change for the next official release which I was planning for sometime this month.  Although looking now more like the end of the month.  

I'm (not) working at the moment, but when I get back home I'll at least touch these up to be consistent.  If you think we should switch to semantic let me know and I'll roll that in as well.
